src/xrGame/ai/stalker/ai_stalker_fire.cpp

```diff
@@ -405,7 +414,22 @@ void CAI_Stalker::update_best_item_info()
 
 void CAI_Stalker::update_best_item_info_impl()
 {
+    luabind::functor<CScriptGameObject*> funct;
+    if (GEnv.ScriptEngine->functor("ai_stalker.update_best_weapon", funct))
```

Comment:
`ai_stalker` module / global object is missing in default gamedata object. For me it caused non-stop exceptions / error logs / drops in performance

Reply:
Will fix it.

Reply:
Fixed in fda9c5e1ffe5e0d9ad7245f22c3505c39d655759.

Reply:
Fixed in 947749cdee95f1dc9b5f251487dcd6929424f161.